File: 0684/sesión15 - consulta/SoftVen/SoftVenModel/ProductosDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftVen.Model
{
    public class ProductosDTO
    {
        private int? productoId;
        private string nombre;
        private string descripcion;
        private double? precio;

        public ProductosDTO()
        {
            this.ProductoId = null;
            this.Nombre = null;
            this.Descripcion = null;
            this.Precio = null;
        }

        public ProductosDTO(int productoId, string nombre, string descripcion, double precio)
        {
            this.ProductoId = productoId;
            this.Nombre = nombre;
            this.Descripcion = descripcion;
            this.Precio = precio;
        }

        public int? ProductoId { get => productoId; set => productoId = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Descripcion { get => descripcion; set => descripcion = value; }
        public double? Precio { get => precio; set => precio = value; }
    }
}

File: 0684/sesión22 - WS - RESTFul/Back-End/ServidorRESTFul/SoftVenModel/src/main/java/pe/edu/pucp/softven/model/ProductosDTO.java
package pe.edu.pucp.softven.model;

public class ProductosDTO {

    private Integer productoId;
    private String nombre;
    private String descripcion;
    private Double precio;

    public ProductosDTO() {
        this.productoId = null;
        this.nombre = null;
        this.descripcion = null;
        this.precio = null;
    }
    
    public ProductosDTO(Integer productoId, String nombre, String descripcion, Double precio) {
        this.productoId = productoId;
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.precio = precio;
    }

    /**
     * @return the productoId
     */
    public Integer getProductoId() {
        return productoId;
    }

    /**
     * @param productoId the productoId to set
     */
    public void setProductoId(Integer productoId) {
        this.productoId = productoId;
    }

    /**
     * @return the nombre
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * @param nombre the nombre to set
     */
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    /**
     * @return the descripcion
     */
    public String getDescripcion() {
        return descripcion;
    }

    /**
     * @param descripcion the descripcion to set
     */
    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    /**
     * @return the precio
     */
    public Double getPrecio() {
        return precio;
    }

    /**
     * @param precio the precio to set
     */
    public void setPrecio(Double precio) {
        this.precio = precio;
    }
}
